src/arns.lua

```diff
@@ -653,6 +704,12 @@ function arns.submitAuctionBid(name, bidAmount, bidder, timestamp, processId, ty
 
 	local finalBidAmount = math.min(requiredOrBidAmount, requiredBid)
```

Comment:
Nit: this doesn't need to be math.min - it's always just the requiredBid.

Reply:
within https://github.com/ar-io/ar-io-network-process/commit/d206577db7372e566ad3aa4ba5493769cff561b5